infra/base-images/base-builder/Dockerfile

```diff
@@ -14,7 +14,8 @@
 #
 ################################################################################
 
-FROM gcr.io/oss-fuzz-base/base-clang
+ARG BASE_IMAGE=gcr.io/oss-fuzz-base/base-clang
```

Comment:
do we need this? 

I think for now, most of our anticipated usages can just rely on base-clang-full directly?

Reply:
Maybe we can figure this out tomorrow, but I think yes, isn't the indexer going to be needed to run in the same image building the project? Maybe this assumption I made is wrong.

Reply:
IMO, we can just pull out the indexer binary and inject it into a regular oss-fuzz project build. IIRC the only runtime dependency it has a LLVM/C headers which are also there in the regular base-clang.

There should be no need to rebase the project image on top of these, so we can keep things simpler.

Reply:
WDYT I wait to use the indexer then before merging either approach? I just don't know if this is feasible. But if you've done it, we can go with the approach your recommending if you think it works.

Reply:
yeah, let's try the static indexer binary approach -- if that fails we can come back to this. We can avoid some complexity this way